javascript-forms/form-values.js

```diff
@@ -0,0 +1,10 @@
+var msgData = {};
```

Comment:
You should move this variable into the `contactFormSubmit` function so it doesn't continue to exist after the function runs.